packages/framework/fluid-static/src/types.ts

```diff
@@ -114,6 +119,14 @@ export interface IRootDataObject {
 	 * @typeParam T - The class of the `DataObject` or `SharedObject`.
 	 */
 	create<T>(objectClass: SharedObjectKind<T>): Promise<T>;
+
+	/**
+	 * Export uploadBlob to outer interface
```

Comment:
Docs like these should generally explain what the method _does_, rather than details like these. If this information is important to include, it should go in a `@privateRemarks` block.